src/main/resources/db/changelog/v.0.2/db.changelog-form.yaml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+  - changeSet:
+      id: 10042021-add-colomn-for-soft-delete-to-form
+      author: nikita_bogurov
+      changes:
+        - addColumn:
+            tableName: form
+            columns:
+              name: fm_deleted
+              type: BOOLEAN
+        - addDefaultValue:
```

Comment:
setup default value inside of addColumn tag

Reply:
Done